### ISSUE TITLE
genesis login fails with bosh cli

### ISSUE THREAD

**Issue by michaelmccaskill** [genesis-community/bosh-genesis-kit#124]
Upgrading from bosh cli 6.4.1 -> 6.4.3 introduces an error:
```
$ genesis -C bosh-deployments do c-ops login

Verifying availability of vault 'c-ops' (https://xxxxx)...ok

Running login addon for c-ops
Logging you in as user 'admin'...
invalid argument for flag `--ca-cert' (expected opts.CACertArg): Expected CA cert to be non-empty

Exit code 1
```

**Comment by thomasmitchell** [comment]
this is because the newest bosh cli doesn't like the BOSH_CA_CERT envvar to be set, but be the empty string. Previously, it treated an empty string as effectively being unset. The addon hook needs to be updated to reflect this new "improved" behavior :)

**Comment by norman-abramovitz** [comment]
pr #125 was created to resolve this issue

**Comment by michaelmccaskill** [comment]
Closed via #125
